staking/effective/eligible.go

```diff
@@ -1,10 +1,22 @@
 package effective
 
-import (
-	staking "github.com/harmony-one/harmony/staking/types"
-)
+// Eligibility represents ability to participate in EPoS auction
+// that occurs just once an epoch on beaconchain
+type Eligibility byte
 
-// IsEligibleForEPOSAuction ..
-func IsEligibleForEPOSAuction(v *staking.ValidatorWrapper) bool {
-	return v.Active && !v.Banned
-}
+const (
+	// Unknown is a default state that represents bad input
+	Unknown Eligibility = iota
```

Comment:
It shouldn't be a bad input. Better call it "Nil" to represent the no-op the nil pointer previously is meant to achieve.